README.md

```diff
@@ -10,7 +10,7 @@ Purely-functional map and set data structures.
 ## Installation
 
 ```
-bower install purescript-ordered-collections
+spago install ordered-collections
 # or
 psc-package install ordered-collections
```

Comment:
This is the only core library mentioning `psc-package` in its installation instructions, should we remove those lines for consistency?

Reply:
Let's remove it.